common/src/main/java/org/opensearch/ml/common/utils/StringUtils.java

```diff
@@ -142,6 +143,27 @@ public static Map<String, String> getParameterMap(Map<String, ?> parameterObjs)
         return parameters;
     }
 
+    @SuppressWarnings("removal")
+    public static LinkedHashMap<String, String> getOrderedMap(Map<String, ?> parameterObjs) {
```

Comment:
Seems this method not being used, remove it?

Reply:
removed.